assets/generated_samples_list.json

```diff
@@ -880,14 +902,28 @@
   },
   "set_basemap": {
     "category": "Maps",
-    "description": "Set a map's basemap. A basemap is beneath all layers on a map and is used to provide visual reference for the operational layers.",
+    "description": "Change a map's basemap. A basemap is beneath all layers on a map and is used to provide visual reference for the operational layers.",
```

Comment:
I think becasue I forgot to update this json after updating the metadata for this sample in the past, it update it now because I ran the script

Reply:
Yes, it's fine to leave them. These changes showed up in other PRs too. Somebody will merge them first, and then everyone will have them and it'll straighten itself out.

Reply:
This will be much smoother when we link up some of these scripts too and automate more, so it doesn't matter if it occasionally gets missed for now!